packages/api/.env.example

```diff
@@ -9,14 +9,14 @@ WEB_URL=http://localhost:3000
 
 CACHE_VIEWS=false
 
-APP_KEY=
+APP_KEY='change this to something else'
 
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
 DB_USER=root
 DB_PASSWORD=
-DB_DATABASE=sabia
+DB_DATABASE=sabia-testing
```

Comment:
Aqui não seria o BD em si? O `sabia-testing` seria o BD pra rodar os testes. Pode ser que acabe confundindo quem for criar o arquivo .env não acha?

Reply:
verdade, vou corrigir.